Resources/Prototypes/Floof/Entities/Clothing/Head/misc.yml

```diff
@@ -19,3 +19,25 @@
     sprite: Clothing/Head/Hats/dogears.rsi
   - type: Clothing
     sprite: Clothing/Head/Hats/dogears.rsi
+
+- type: entity
+  parent: ClothingHeadBase
+  id: ClothingHeadSexCompCirclet
+  name: sex competition circlet
+  description: It looks like it should be part of a competition.
+  components:
+  - type: Sprite
+    sprite: Floof/Clothing/Head/sexcompcirclet.rsi
+  - type: Clothing
+    sprite: Floof/Clothing/Head/sexcompcirclet.rsi
+
+- type: entity
+  parent: ClothingHeadHatHoodWinterBase
+  id: ClothingHeadHatHoodMarsHoodieHood
+  noSpawn: true
+  name: mars hoodie hood
+  components:
+  - type: Sprite
+    sprite: Floof/Clothing/Head/hoodmarshoodie.rsi
+  - type: Clothing
+    sprite: Floof/Clothing/Head/hoodmarshoodie.rsi

```

Comment:
Mars Hoodie Hood?
This look like a self-insert and will need approval of @Memeji

Reply:
Understood!~ It was a request from someone. I can easily change it to "Pink Hoodie" if it would work better.

Reply:
Its would, example on EE i have the "Office Vulp Plushie".
but yeah, those look like personnel items, in all honest best is always having approval from Memeji.

Reply:
Okay! I'll change it up to be more generic! Should I be changing the RSI file names as well? First time doing this, so was unaware that we need approval for personal items.

Reply:
rsi file names can stay.

yeah but best to ask.